src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise-detail.component.ts

```diff
@@ -71,12 +81,37 @@ export class TextExerciseDetailComponent implements OnInit, OnDestroy {
             this.formattedGradingInstructions = this.artemisMarkdown.safeHtmlForMarkdown(this.textExercise.gradingInstructions);
             this.formattedProblemStatement = this.artemisMarkdown.safeHtmlForMarkdown(this.textExercise.problemStatement);
             this.formattedExampleSolution = this.artemisMarkdown.safeHtmlForMarkdown(this.textExercise.exampleSolution);
+            this.detailOverviewSections = this.getExerciseDetailSections();
         });
         this.statisticsService.getExerciseStatistics(exerciseId).subscribe((statistics: ExerciseManagementStatisticsDto) => {
             this.doughnutStats = statistics;
         });
     }
 
+    getExerciseDetailSections() {
```

Comment:
```suggestion
    private getExerciseDetailSections() {
```

This can be private.

Reply:
see above